configs/pantheon.json

```diff
@@ -16,13 +16,14 @@
     "lvl2": "#doc h3",
     "lvl3": "#doc h4",
     "lvl4": "#doc h5",
-    "text": "#doc p, #doc li"
+    "text": "#doc p, #doc li, #doc ol, #doc code, #doc alert"
```

Comment:
We don't recommend to extract those elements since they will include noise and duplicates (`ol` encompasses `li`)
`code` element mostly introduce noise. 
What about adding a more specific classe?

Reply:
Thanks @s-pace! I updated the PR to add only our notes and warnings to the text selector.

Reply:
👍